src/host/main.cpp

```diff
@@ -388,6 +388,14 @@ int main(int argc, char** argv)
     ->transform(CLI::CheckedTransformer(curve_id_map, CLI::ignore_case))
     ->capture_default_str();
 
+  uint32_t node_id = -1;
+  app
+    .add_option(
+      "--bft-node-id",
```

Comment:
I don't think this should be assigned manually, there's a serious re-use risk. We should work this out from the public ledger recovery instead.

Reply:
```
      auto tx = network.tables->create_tx();
      GenesisGenerator g(network, tx);
      g.create_service(network.identity->cert);
      g.retire_active_nodes();

      g.add_node(
```

I think we can set self correctly in the gap here, and have a temp value before.